site/app/controllers/PollController.php

```diff
@@ -503,6 +505,88 @@ public function openPoll(): RedirectResponse {
     }
 
     /**
+     * @Route("/courses/{_semester}/{_course}/polls/addCustomResponse", methods={"POST"})
```

Comment:
Should these be php8 routes now that we have refactored a lot of the phpdoc routes?

Reply:
[a6735ec](https://github.com/Submitty/Submitty/pull/10197/commits/a6735ec8ff21da1dd55256759150bc150e2fd47f)